corehq/apps/smsforms/app.py

```diff
@@ -120,6 +120,8 @@ def submit_unfinished_form(session):
     # Get and clean the raw xml
     try:
         response = FormplayerInterface(session.session_id, session.domain).get_raw_instance()
+        if response.get('status') == 'error':
+            raise TouchformsError(response.get('exception'))
```

Comment:
In the error we saw recently (the one with the "-1"), are you sure there's a `"status": "error"`? I remembered there not being but I'm probably wrong. It might be helpful to the next person coming along to have a comment here that has what it expects `response` to look like in this situation.

Reply:
It does include status, looks like all formplayer responses do. Added 356e1de147e to comment on the error info.